Events/2022-05-ECP22-BOF.md

```diff
@@ -0,0 +1,55 @@
+# 2022 Exascale Computing Project Community BOF Days
+
+#### Contributed by BSSw.io Team
+
+- Date: May 10 - 12, 2022
+- Location: Online
+- Event Website: https://www.exascaleproject.org/event/ecp-community-bof-days-2022/
+- Organizers: Exascale Computing Project Community
+
+The Exascale Computing Project's Birds-of-a-Feather Days provide an opportunity for the High Performance Computing community to engage with their peers in various sessions to discuss current development efforts in the exascale computing domain.
+
+Event Information | Details
+:--- | :---			   
+Event Name | [ECP BOF Days](https://www.exascaleproject.org/event/ecp-community-bof-days-2022/) 
+Event Date & Time | May 10 - 12, 2022 (60- to 90-minute sessions throughout the day)
+Website | 	<https://www.exascaleproject.org/event/ecp-community-bof-days-2022/>  
+
+The Exascale Computing Project (ECP) 2022 Community Birds-of-a-Feather (BOF) Days will take place Tuesday, May 10 – Thursday, May 12, with six to eight sessions per day, similar to the 2021 event. 
+The 2022 Community BOF Days will provide an opportunity for the high-performance computing community to engage with ECP teams to discuss our latest development efforts. 
+Each BOF will be a 60- to 90-minute session on a given topic, with a brief overview, followed by Q&A. 
+The BOFs will be conducted via Zoom.
+
+Topics, listed below, will range from (**please put information here**), as well as general topics focusing on software sustainability and productivity, approaches to software quality assurance, etc. 
```

Comment:
I don't want to try to summarize the other topics in a couple of words -- too diverse.  Instead, I suggest switching it around to say "BOFs will include general topics focusing on... as well as a range of others related to ECP projects and acivities."